test/acceptance/README.md

```diff
@@ -9,7 +9,7 @@ To run tests you need to have the following installed:
   ```bash
   $ wget -O- https://k14s.io/install.sh | bash
   ```
-  
+
```

Comment:
👾 👾 👾 👾 

Probably fine tho

Reply:
good catch!!! :grinning:

Reply:
These spaces were there originally for what it's worth. We just removed the trailing whitespace. 😅 

https://raw.githubusercontent.com/cloudfoundry/cf-k8s-networking/179bf56502986aab27a98df8f7f33b0627e564e5/test/acceptance/README.md